docs/output.md

```diff
@@ -2,58 +2,265 @@
 
 ## Introduction
 
-This document describes the output produced by the pipeline. Most of the plots are taken from the MultiQC report, which summarises results at the end of the pipeline.
+This document describes the output produced by the pipeline based on public data.
```

Comment:
I guess you mean here the aws full size test? Somehow this sentence makes me think that with other data there could be more plots that are not explained. Is that the case

Reply:
Public data here is the GM12878 used as sample input, not aws. Although the output directory and files are specified, plos as such are not given, and I believe the plots are mainly the QC plots at read/mapping level. Can I add this later?